pr_watch/migrations/0009_organization_db_index.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
```

Comment:
@rocioar Would it be possible to merge this migration into the previous one? (`update_organization`)?

Reply:
@itsjeyd actually this was done because there was an issue with postgres when changing the type of a db_index from a char to an int, see: https://code.djangoproject.com/ticket/27860

Reply:
@rocioar Ah, OK. Totally fine to keep both of these migrations, then :)